internal/apps/ai-proxy/filters/azure-director/filter.go

```diff
@@ -127,42 +127,42 @@ func (f *AzureDirector) FindProcessor(ctx context.Context, processor string) (fu
 func (f *AzureDirector) DoNothing(context.Context) error { return nil }
 
 func (f *AzureDirector) SetAuthorizationIfNotSpecified(ctx context.Context) error {
-	value, ok := ctx.Value(reverseproxy.CtxKeyMap{}).(*sync.Map).Load(vars.MapKeyProvider{})
+	value, ok := ctx.Value(reverseproxy.CtxKeyMap{}).(*sync.Map).Load(vars.MapKeyModelProvider{})
 	if !ok || value == nil {
 		return errors.New("provider not set in context map")
 	}
-	prov := value.(*models.AIProxyProviders)
+	prov := value.(*model_provider.ModelProvider)
 	reverseproxy.AppendDirectors(ctx, func(req *http.Request) {
-		if appKey := prov.GetAPIKeyWithDecrypt(); appKey != "" && req.Header.Get("Authorization") == "" {
-			req.Header.Set("Authorization", vars.ConcatBearer(appKey))
+		if apiKey := prov.APIKey; apiKey != "" && req.Header.Get("Authorization") == "" {
+			req.Header.Set("Authorization", vars.ConcatBearer(apiKey))
```

Comment:
Is apiKey need encrypted in header Authorization？

Reply:
No, the same as OpenAI and Azure OpenAI Service.